src/vs/platform/commands/common/commands.ts

```diff
@@ -122,10 +120,13 @@ export const CommandsRegistry: ICommandRegistry = new class implements ICommandR
 	}
 
 	getCommands(): ICommandsMap {
-		const result: ICommandsMap = Object.create(null);
-		this._commands.forEach((value, key) => {
-			result[key] = this.getCommand(key)!;
-		});
+		const result = new Map<string, ICommand>();
```

Comment:
nit: Same thing here - could you just return `new Map<string, ICommand>(this._commands)`?